build.sbt

```diff
@@ -25,7 +25,7 @@ resolvers += "ATS Snapshots" at "http://nexus.advancedtelematic.com:8081/content
 libraryDependencies ++= {
   val akkaV = "2.5.13"
   val akkaHttpV = "10.1.1"
-  val libatsV = "0.2.1-13-g4a34692"
+  val libatsV = "0.2.1-18-g7726a72"
```

Comment:
Does the version of `libats` exists?

Reply:
Nope, it's the one I published locally with changes from advancedtelematic/libats#95 because that PR is not merged yet.

Reply:
I think the original idea of @jerrytrieu was to figure out whether it is possible to switch campaigner <-> director comms to messages and then merge advancedtelematic/libats#95 when it's done.